SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -198,11 +266,9 @@ private static void TryFixSarifReport(ILogger logger, ProjectInfo project, IRosl
                     if (fixedPath != null)
                     {
                         // otherwise, set the property value (results in no change if the file was already valid)
-                        Property newReportPathProperty = new Property
-                        {
-                            Id = reportFilePropertyKey,
-                            Value = fixedPath
-                        };
+                        Property newReportPathProperty = new Property();
```

Comment:
Why this change? The initializer looks better.

Reply:
Probably I missed that... I copy-pasted my files on top of JH's changes and then manually reverted changes.